### ISSUE TITLE
Form Number Input .valid 

### ISSUE THREAD

**Issue by UWouldlLose** [angular/angular#5507]
it looks like numeric input fields don't accept 0 as a valid value. 

I have the following markup: 

    <div class="form-group">
        <label for="reportingDigits">Reporting Digits</label>
            <input type="number" class="form-control" name="reportingDigits" required maxlength="3"
                   [(ng-model)]="plrTest.ReportingDigits"
                   ng-control="reportingdigits" #reportingdigits="form">

            <div [hidden]="reportingdigits.valid" class="alert alert-danger">
                Reporting Digits is required.
            </div>
        </div>

The validation div shows up, but the form is allowed to submit.

**Comment by PascalPrecht** [comment]
Hey @UWouldlLose 

It's quite hard to follow what you're trying to achieve based on your description. Can you setup a plunk that reproduces what you think doesn't work?

**Comment by UWouldlLose** [comment]
Take a look here: http://plnkr.co/edit/uijHwN9Fe30jS12VFqfH?p=preview

If you put 0 in the input box, validation doesn't pass. 

I would expect that 0 is a valid numeric value.

**Comment by ericmartinezr** [comment]
@UWouldlLose this issue was fixed in beta.6
